projectDocs/Design_Implementation.md

```diff
@@ -0,0 +1,54 @@
+# Design (Subject to updates)
+## Model  
+### Database  
+|   Database column   |   expected input   |        Thoughts on implementation        |
+| ------------------- | ------------------ | ---------------------------------------- |
+|          pk         |    auto-generate   |                                          |
+|       subject       |  string from user  | short description to user to add clarity |
+|  Issue comment FK   |  string from user  | long description to user to add clarity  |
```

Comment:
This is not a string. If memory serves, this FK (foreign key) identifies the record in the comment table that serves as the long description. So this is a number; one of the comment primary key values. This is mirroring the structure of GitHub Issues, where creating an Issue involves creating the first of potentially many comments.